### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by yadaiio**
Builds on top of #30, #32 and #33.

References: https://github.com/clue/reactphp-tar/pull/20, https://github.com/clue/reactphp-redis/pull/117, https://github.com/clue/reactphp-shell/pull/20 and https://github.com/clue/reactphp-sqlite/pull/43.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ element may be broken up into multiple chunks.
 This class reassembles these elements by buffering incomplete ones.
 
 The `Decoder` supports the same optional parameters as the underlying
-[`str_getcsv()`](http://php.net/str_getcsv) function.
+[`str_getcsv()`](https://php.net/str_getcsv) function.
```
Comment (clue) [inline README.md]: Should probably be replaced with canonical link, WDYT? 👍

```suggestion
[`str_getcsv()`](https://www.php.net/manual/en/function.str-getcsv.php) function.
```

Comment (yadaiio) [reply README.md]: Oops, my fault to not look closer to the link. Thank you @clue for noticing this.

**Review by SimonFrings — APPROVED** [review]
Looks good to me :+1:

File: README.md [hunk 2]
```diff
@@ -173,7 +173,7 @@ element may be broken up into multiple chunks.
 This class reassembles these elements by buffering incomplete ones.
 
 The `Decoder` supports the same optional parameters as the underlying
-[`str_getcsv()`](http://php.net/str_getcsv) function.
+[`str_getcsv()`]((https://www.php.net/manual/en/function.str-getcsv.ph) function.
```
Comment (clue) [inline README.md]: This doesn't look right:

```suggestion
[`str_getcsv()`](https://www.php.net/manual/en/function.str-getcsv.php) function.
```

Comment (yadaiio) [reply README.md]: Opps, I copied it wrong, thank you for noticing this @clue. Should be fixed now. 👍

**Review by clue — APPROVED** [review]
@yadaiio Thanks for the update, changes LGTM! :shipit:

**Pull request merged**